.github/workflows/dev-CD.yml

```
@@ -65,4 +64,69 @@ jobs:
           username: ${{ secrets.DEV_SERVER_USERNAME }}
           key: ${{ secrets.DEV_SERVER_KEY }}
           script: |
-            bash /home/ubuntu/Napzak-BE/deploy.sh
\ No newline at end of file
+            cd /home/ubuntu/Napzak-BE
+            git fetch origin develop && git reset --hard origin/develop
+            bash deploy.sh
+  yaml
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 70: could not find expected ':'

(syntax-check)